Source/EasyNetQ/EasyNetQ.csproj

```diff
@@ -38,8 +38,8 @@
     <Reference Include="Newtonsoft.Json">
       <HintPath>..\packages\Newtonsoft.Json.7.0.1\lib\net40\Newtonsoft.Json.dll</HintPath>
     </Reference>
-    <Reference Include="RabbitMQ.Client">
-      <HintPath>..\packages\RabbitMQ.Client.3.5.5\lib\net40\RabbitMQ.Client.dll</HintPath>
+    <Reference Include="RabbitMQ.Client, Version=3.5.6.0, Culture=neutral, PublicKeyToken=89e7d7c5feba84ce, processorArchitecture=MSIL">
```

Comment:
I got rid of that manually on the previous pull. Not sure if it has any influence though (I thought it would maybe not require you to use redirects when using mixed versions, or would it still get this info from the assembly on compilation?)